wagtail/images/models.py

```diff
@@ -1408,6 +1408,62 @@ def background_position_style(self):
         else:
             return "background-position: 50% 50%;"
 
+    @property
+    def background_position_x(self):
+        """
+        Returns the horizontal background position as a percentage string.
+
+        This positions the rendition horizontally according to the focal point's x coordinate.
+        If no focal point is set, defaults to 50% (center).
+
+        Returns:
+            str: The horizontal position as a percentage (e.g., "25%")
+
+        This can be passed as a data attribute to client-facing HTML.
+
+        A usage example in the client-facing CSS (to avoid inline styles):
+            <div class="my-bg-image" data-background-position-x="{{ image.background_position_x }}">
+            </div>
+
+            .my-bg-image {
+                background-position-x: attr(data-background-position-x, <number>);
+            }
```

Comment:
```suggestion
        This can be passed as a data attribute to then be used in JavaScript to set the `background-position-x` CSS property instead of using inline styles.

            <div class="my-bg-image" data-background-position-x="{{ image.background_position_x }}">
            </div>
```